CHANGES.md

```diff
@@ -1,6 +1,417 @@
+* added 'param' and 'type' overloads of getField() to the Reflection module
+  (see http://chapel.cray.com/docs/master/modules/standard/Reflection.html?highlight=getfield#Reflection.getField)
+* added a new lines() iterator to channels
+  (see http://chapel.cray.com/docs/master/modules/standard/IO.html?highlight=lines#IO.channel.lines)
+* added support for file.getParentName() to the `Path` module
+  (see http://chapel.cray.com/docs/master/modules/standard/Path.html#Path.file.getParentName)
+* added ctime() to the 'datetime' type in the 'DateTime' module
+  (see http://chapel.cray.com/docs/master/modules/standard/DateTime.html?highlight=ctime#DateTime.datetime.ctime)
+* added guided and adaptive iterators that accept domains
+  (see http://chapel.cray.com/docs/latest/modules/standard/DynamicIters.html)
+* gave the dynamic() iterators a default chunk size of 1
+* added an asciiToString function that converts a uint(8) into a string
+  (see http://chapel.cray.com/docs/latest/builtins/internal/String.html?highlight=asciitostring#String.asciiToString)
+* made the printing of comm diagnostics in 'CommDiagnostics' suppress zeroes
+* closed memory leaks in regular expressions / RegEx
+
+Standard Domain Maps (Layouts and Distributions)
+------------------------------------------------
+* generalized LayoutCSR to LayoutCS to support CSR and CSC sparse layouts
+  (see http://chapel.cray.com/docs/master/modules/layouts/LayoutCS.html)
+* made the ReplicatedDist distribution less odd and renamed it to 'Replicated'
+  (see http://chapel.cray.com/docs/master/modules/dists/ReplicatedDist.html
+   and http://chapel.cray.com/docs/master/primers/replicated.html)
+* added a 'replicand' method to 'ReplicatedDist' supporting local access
+  (see http://chapel.cray.com/docs/master/modules/dists/ReplicatedDist.html?highlight=replicand
+   and http://chapel.cray.com/docs/master/primers/replicated.html?highlight=replicand)
+* privatized sparse Block-distributed domains/arrays
+* closed memory leaks in the DimensionalDist2D distribution
+
+Package Modules
+---------------
+* added support for a new 'Crypto' module based on OpenSSL
+  (see http://chapel.cray.com/docs/master/modules/packages/Crypto.html)
+* added a new 'Collections' module  (TODO: also DistributedBag/Dequeue?)
+  (see TODO)
+* added support for a new 'TOML' module (not 100% feature complete)
+  (see TODO)
+* added support for a distributed guided and dynamic iterators
+  (see http://chapel.cray.com/docs/master/modules/packages/DistributedIters.html)
+* improved support for the MPI module when ugni, gasnet/aries, and/or qthreads
+* improved Chapel-Python interoperability for the 'ZMQ' (ZeroMQ) module
+* improved the 'LinearAlgebra' module in a number of ways:
+  - added a new 'Sparse' sub-module to 'LinearAlgebra'
+    (see http://chapel.cray.com/docs/master/modules/packages/LinearAlgebra/Sparse.html)
+  - added support for arbitrary lower bounds
```

Comment:
This bullet isn't ringing any bells at the moment, unless you're referring to #7057 which was a follow-up to the bullet below. If that's the case, I'd suggest omitting this.

Reply:
That is the PR that triggered it, but I missed that it was diag()-specific.  I'll pull it, thanks!